pkg/fleet/installer/packages/pkg_manager.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+package packages
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// removeDebRPMPackage removes a package installed via deb/rpm package manager
+// It doesn't remove dependencies or purge as we want to keep existing configuration files
+// and reinstall the package using the installer.
+func removeDebRPMPackage(ctx context.Context, pkg string) (err error) {
+	span, _ := tracer.StartSpanFromContext(ctx, "remove_deb_rpm_package")
+	defer func() { span.Finish(tracer.WithError(err)) }()
+	// Compute the right command depending on the package manager
+	var cmd *exec.Cmd
+	if _, pathErr := exec.LookPath("dpkg"); pathErr == nil {
+		cmd = exec.Command("dpkg", "-r", agentPackage)
```

Comment:
I assume we're not querying dpkg because it always succeeds on a remove even if the package is not installed ?

Reply:
Correct